man/sd_bus_set_description.xml

```diff
@@ -0,0 +1,185 @@
+
+      <funcprototype>
+        <funcdef>int <function>sd_bus_set_allow_interactive_authorization</function></funcdef>
+        <paramdef>sd_bus *<parameter>bus</parameter></paramdef>
+        <paramdef>int <parameter>b</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>sd_bus_get_allow_interactive_authorization</function></funcdef>
+        <paramdef>sd_bus *<parameter>bus</parameter></paramdef>
+      </funcprototype>
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_bus_set_description()</function> sets the description string
+    that is used in logging to the specified string. The string is copied internally
+    and freed when the bus object is deallocated. The
+    <parameter>description</parameter> argument may be <constant>NULL</constant>, in
+    which case the description is unset. This function must be called before the bus
+    has been started.</para>
+
+    <para><function>sd_bus_get_description()</function> returns a description string
+    in <parameter>description</parameter>. This string may have been previously set
+    with <function>sd_bus_set_description()</function> or
+    <citerefentry><refentrytitle>sd_bus_open_with_description</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    or similar. If not set this way, a default string like <literal>system</literal>
+    or <literal>user</literal> will be returned for the system or user buses,
+    and <constant>NULL</constant> otherwise.</para>
+
+    <para><function>sd_bus_set_anonymous()</function> enables or disables "anonymous
+    authentication", i.e. lack of authentication, of the bus peer. This function must
+    be called before the bus has been started. See the <ulink
+    url="view-source:https://dbus.freedesktop.org/doc/dbus-specification.html#auth-mechanisms">Authentication
+    Mechanisms</ulink> section of the D-Bus specification for details.</para>
+
+    <para><function>sd_bus_set_trusted()</function> sets the "trusted" state on the
+    <parameter>bus</parameter> object. If true, all connections on the bus are
+    trusted and access to all privileged and unprivileged methods is granted.  This
+    function must be called before the bus has been started.</para>
+
+    <para><function>sd_bus_set_allow_interactive_authorization()</function> enables
+    or disables interactive authorization for method calls. If true, messages are
+    marked with the <constant>ALLOW_INTERACTIVE_AUTHORIZATION</constant> flag
+    specified by the <ulink
+    url="view-source:https://dbus.freedesktop.org/doc/dbus-specification.html">D-Bus</ulink>
+    specification, informing the receiving side that the caller is prepared to wait
+    for interactive authorization, which might take a considerable time to
+    complete. If this flag is set, the user may be queried for passwords or
+    confirmation via Polkit or a similar framework.</para>
```

Comment:
Polkit → PolicyKit. We use "polkit" only as sloppy, internal name. But I think userfacing docs should use the full, official name of the package.

Reply:
It seems that Policykit was renamed to polkit in 2012, see https://cgit.freedesktop.org/polkit/tree/NEWS#n754. Their public documentation consistently uses polkit. I think we need to follow suit. I'll push another patch to do the renaming in our documentation.